core/src/main/java/io/neow3j/protocol/OfflineService.java

```diff
@@ -0,0 +1,46 @@
+package io.neow3j.protocol;
+
+import io.neow3j.protocol.core.Request;
+import io.neow3j.protocol.core.Response;
+import io.neow3j.protocol.exceptions.OfflineServiceException;
+import io.neow3j.protocol.notifications.Notification;
+import io.reactivex.Observable;
+
+import java.io.IOException;
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * An implementation of the Service API that restricts using any of its functions and throws appropriate exceptions
+ * if they are used.
+ */
+public class OfflineService implements Neow3jService {
```

Comment:
What about javadocs?! 👀

Reply:
I can add some, sure. We don't have any on these functions covered with javadoc btw, neither in its counterpart class `Service.java` (which is extended by HttpService and HttpService does not override Service's send function for example), nor the more frequently used functions in `Request.java` and the code there is ~7 years old. Won't mention the blame 😛😉

https://github.com/neow3j/neow3j/blob/fe11c08fca3031d2c1aa11163d7bef210b44dda1/core/src/main/java/io/neow3j/protocol/core/Request.java#L68-L78

https://github.com/neow3j/neow3j/blame/fe11c08fca3031d2c1aa11163d7bef210b44dda1/core/src/main/java/io/neow3j/protocol/Service.java#L48-L71

I will add some proper javadoc in both places. 😄

Reply:
I know, I know.

But we should have the mentality to improve code standards bit by bit, in every PR. That's why I asked.